db/create_tables.sql

```diff
@@ -1,4 +1,4 @@
-CREATE TABLE IF NOT EXISTS `audit` (
+CREATE TABLE IF NOT EXISTS `audit_records` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `state_machine_name` VARCHAR(255) NOT NULL,
   `state_machine_instance_id` VARCHAR(255) NOT NULL,
```

Comment:
should we have a created_at column over here?
Also, do we have one row per state of a state machine or 1 row for everything that happens to a state?

For eg: After successful execution of a state, will the table read
<sm_id,state_id,retryAttempt:1,state_status:Completed,..)
OR
<sm_id,state_id,retryAttempt:0,state_status:Failed,..)
<sm_id,state_id,retryAttempt:1,state_status:Completed)